### DISCUSSION TITLE
Nothing happened when I tried to add my first card

### DISCUSSION THREAD

**Discussion by cwLearning** [st3v3nmw/obsidian-spaced-repetition#490] [Q&A]
I installed this plugin directly from the Obsidian app, using a VPN.  I'm sure that I have enabled this plugin after installation. No matter what separators I used(:: ::: ?? ???), nothing happened. I can't create either flashcards or decks. The number of cards has never changed. 
I wonder if this is a bug or if I have misunderstood the instructions of this plugin. Could anybody help me? Thanks.
![Snipaste_2022-08-31_14-30-32](https://user-images.githubusercontent.com/112560175/187688792-e190cccc-2ff3-4053-9147-d2dbf683ac8b.png)

**Comment by st3v3nmw** [comment]
Hi @cwLearning, how do your settings look like? You should either have added a tag that you're using to tag your cards or enabled the options that auto-converts the folder structure to decks & subdecks
